java/HealthVault Mobile/HVSample/src/main/res/values/dimens.xml

```diff
@@ -4,81 +4,121 @@
 	<dimen name="activity_horizontal_margin">16dp</dimen>
 	<dimen name="activity_vertical_margin">16dp</dimen>
 
-
-	<!--Home Page -->
-
 	<!--Weight Page -->
-	<dimen name="marginleft_20dp">20dp</dimen>
-	<dimen name="marginright_20dp">20dp</dimen>
-	<dimen name="margintop_10dp">10dp</dimen>
-	<dimen name="margintop_20dp">20dp</dimen>
-	<dimen name="marginbottom_10dp">10dp</dimen>
-
-	<dimen name="weightlayout_marginleft">20dp</dimen>
-	<dimen name="weightlayout_marginright">200dp</dimen>
-	<dimen name="weightlayout_height_40dp">40dp</dimen>
-	<dimen name="weightlayout_height_20dp">20dp</dimen>
-
-	<dimen name="weight_graphcontainer_height">237dp</dimen>
-	<dimen name="weight_graphcontainer_width">360dp</dimen>
-
-	<dimen name="lastweightlabel_width">75dp</dimen>
-	<dimen name="lastweightlabel_height">20dp</dimen>
-	<dimen name="lastweightlabel_marginleft">20dp</dimen>
-	<dimen name="lastweightlabel_margintop">26dp</dimen>
-
-	<dimen name="lastweighttext_width">80dp</dimen>
-	<dimen name="lastweighttext_height">62dp</dimen>
-	<dimen name="lastweighttext_marginleft">20dp</dimen>
-	<dimen name="lastweighttext_marginright">20dp</dimen>
-	<dimen name="lastweighttext_marginbottom">20dp</dimen>
-
-	<dimen name="lastweightmetric_width">16dp</dimen>
-	<dimen name="lastweightmetric_height">16dp</dimen>
+	<dimen name="horizontal_margin_medium">20dp</dimen>
```

Comment:
We still have a ton of dimens with the same exact value. Our names are still quite specific, for example "weight_layout_margin_left" isn't very generic and denotes a specific use. For the app we used to do things like "MediumMargin = 20dp", "SmallMargin = 10dp" etc. We should really be using generic values like that to have uniform styling throughout the app. I get this is a sample though and we might not actually care... but we should also be mindful of the fact that all code we write in these public repos should be a strong example of how code should be written